src/main/java/org/traccar/protocol/HuabaoProtocolDecoder.java

```diff
@@ -699,6 +700,18 @@ private Position decodeLocation(DeviceSession deviceSession, ByteBuf buf) {
                         }
                     }
                     break;
+                case 0xF4:
+                    Network network = new Network();
```

Comment:
I think we need to move `Network` to the top level somewhere, so that we don't accidentally override it.